packages/react-components/react-breadcrumb-preview/stories/BreadcrumbDivider/BreadcrumbSlashDivider.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { Breadcrumb, BreadcrumbItem, BreadcrumbDivider } from '@fluentui/react-breadcrumb-preview';
+
+export const BreadcrumbSlashDivider = () => (
+  <Breadcrumb aria-label="Breadcrumb example with slash divider" size="small" dividerType="slash">
```

Comment:
Should there be a story for custom dividers?

Reply:
No, because it's not recommended to use custom dividers at all. Designers specifically asked me to remove those examples. Slash divider is considered as a custom one